src/lib.rs

```diff
@@ -251,6 +251,31 @@ impl<P: AsRef<[u8]>, T: Transitions> AcAutomaton<P, T> {
                 .map(|s| mem::size_of::<State<T>>() + s.heap_bytes())
                 .sum::<usize>() + self.start_bytes.len()
     }
+
```

Comment:
Remove this extra line please. :-) Should be at most one empty line between any item.

Reply:
Fixed, (I have become reliant on rustfmt removing those, not sure why it didn't in this case).

Reply:
Thanks! Can you squash this down into a previous commit? I can do it if you want, but I think this is the only thing preventing my from just rebasing your commits on to master as-is (which is very convenient, thank you!).

Reply:
Done!